core/src/main/java/com/fairyband/soak/core/designsystem/dialog/BaseDialog.kt

```diff
@@ -0,0 +1,45 @@
+package com.fairyband.soak.core.designsystem.dialog
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+import androidx.compose.ui.window.DialogProperties
+
+@Composable
+fun BaseDialog(
+    modifier: Modifier = Modifier,
+    onDismiss: () -> Unit,
+    horizontalAlignment: Alignment.Horizontal = Alignment.CenterHorizontally,
+    content: @Composable ColumnScope.() -> Unit,
```

Comment:
```suggestion
fun BaseDialog(
    onDismiss: () -> Unit,
    modifier: Modifier = Modifier,
    horizontalAlignment: Alignment.Horizontal = Alignment.CenterHorizontally,
    content: @Composable ColumnScope.() -> Unit,
```
아마 Modifier가 옵셔널 아래로 오는 게 관례였던 것 같아서..!